### DISCUSSION TITLE
IAM Roles Anywhere java code

### DISCUSSION THREAD

**Discussion by fykidwai** [aws/aws-sdk-java#2806] [Q&A]
Is there any blog or coding example available for using [IAM Roles Anywhere](https://docs.aws.amazon.com/general/latest/gr/rolesanywhere.html) through aws-sdk-java?

Also steps detailing all the steps for certificate creation will be great..!

**Comment by yasminetalby** [comment]
Hello @fykidwai ,

Thank you very much for your message.
The [AWS Docs SDK Examples repository](https://github.com/awsdocs/aws-doc-sdk-examples) provides examples on how to use the various SDKs that AWS provides for interacting with its services.
Unfortunately, there is currently no coding example available for using [IAM Roles Anywhere](https://docs.aws.amazon.com/general/latest/gr/rolesanywhere.html) through AWS SDK Java v1.
You can request documentation by submitting a **New Code Example Request** under [New Issues](https://github.com/awsdocs/aws-doc-sdk-examples/issues/new/choose).

Best regards,

Yasmine
